outline.md

```diff
@@ -0,0 +1,17 @@
+# Outline
+
+This document serves to outline the basic structure of the document. It is subject to change.
+
+Suggestions appreciated.
+
+## Faker in the past & motivations
+
+This section will discuss the rise of software testing, how faker was originally concieved to assist with testing, and some of the design decisions in previous versions of faker.
+
+## Death of Faker PHP
+
+Here we will highlight the rise of CI/CD and containerization, and how in this new environment, PHP faker crumbled.
```

Comment:
Currently there is a question in the Faker discord about the future of faker- update once responded to